### PR TITLE
make the continents list fetched from graphql api

### PR DESCRIPTION
**Pull request by thirsty-wing**

### CODE REVIEW

File: utils/hooks/useFetchEarth.js
Comment (thirsty-wing) [inline utils/hooks/useFetchEarth.js:1]: Not sure how to go about testing this file and all files that use it.

File: utils/hooks/useFetchEarth.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ *
+ * @param {string} query in graphql to be fetch the data
+ */
+export function useFetchEarth(query) {
+
+  const [data, setData] = useState({});
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  async function fetchData() {
```
Comment (thirsty-wing) [inline utils/hooks/useFetchEarth.js:19]: Not sure how to get this function in code coverage

File: components/HomeScreen/Title/Title.utils.js [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
  *
- * @param {number} globeOrientation
- * @param {function} setGlobeOrientation
+ * @param {object} args
+ * @param {number} args.globeOrientation
+ * @param {function} args.setGlobeOrientation
+ * @returns {rotateGlobe}
  */
-export function yieldRotateGlobe(globeOrientation, setGlobeOrientation) {
+export function yieldRotateGlobe(args) {
```
Comment (thirsty-wing) [inline components/HomeScreen/Title/Title.utils.js:14]: See this on discussion of this function's name https://github.com/thirsty-wing/earth-app/pull/2#discussion_r1297770283

File: components/HomeScreen/ContinentsList/ContinentsList.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  } = props;
+
+  return(
+    <View
+      style={{
+        flex: 1,
+        gap: theme.spacing(),
+      }}
+    >
+      { loading ? (
```
Comment (bc-bane) [inline components/HomeScreen/ContinentsList/ContinentsList.jsx]: same feedback here as the previous commit. Nest ternaries cause readability issues, I'd consider doing it a different way

Comment (thirsty-wing) [reply components/HomeScreen/ContinentsList/ContinentsList.jsx]: Done.

File: utils/hooks/useFetchEarth.js (hunk 1 above)
Comment (bc-bane) [reply utils/hooks/useFetchEarth.js:19]: Man I'm sorry I missed this comment as well. did you find a way to test it? if not let's go over it together. Can you put a short block on my calendar?

**Pull request merged**